apps/frontend/src/modules/education/pages/EducationKeyboardsPage.css

```
@@ -0,0 +1,29 @@
+  align-items: center;
+}
+
+.keyboards-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  gap: 1rem;
+  margin-top: 1rem;
+  padding: 1rem;
+  justify-content: center;
+  align-items: start;
+  place-items: center;
```

Biome (2.1.2)

[error] Unexpected shorthand property place-items after align-items



(lint/suspicious/noShorthandPropertyOverrides)